src/ua-parser.js

```diff
@@ -448,7 +448,7 @@
             ], [MODEL, [VENDOR, 'Amazon'], [TYPE, TABLET]], [
             /(sd|kf)[0349hijorstuw]+(\sbuild\/|\)).+silk\//i                    // Fire Phone
             ], [[MODEL, mapper.str, maps.device.amazon.model], [VENDOR, 'Amazon'], [TYPE, MOBILE]], [
-            /android.+aft([bms])\sbuild/i                                       // Fire TV
+            /android.+aft([\w])(\sbuild\/|\))/i                                 // Fire TV
```

Comment:
https://developer.amazon.com/docs/fire-tv/user-agent-strings.html
> with a device model that starts with "AFT".

According to the document, It is correct to judge by any single character following "AFT"

Reply:
Supports both with and without "build"